relay-config/src/config.rs

```diff
@@ -1487,8 +1569,11 @@ impl Config {
     ///
     /// This is `true` either if `outcomes.emit_outcomes` is explicitly enabled, or if this Relay is
     /// in processing mode.
-    pub fn emit_outcomes(&self) -> bool {
-        self.values.outcomes.emit_outcomes || self.values.processing.enabled
+    pub fn emit_outcomes(&self) -> EmitOutcomes {
+        if self.processing_enabled() {
+            return EmitOutcomes::AsOutcomes;
+        }
+        self.values.outcomes.emit_outcomes
```

Comment:
@jjbayer this makes me think, maybe we should make the field in the config `Option<EmitOutcomes>` so explicitly setting the value to anything takes precedence over this. but not entirely sure. can leave it out for now